src/components/tools/tool-tile.tsx

```diff
@@ -386,14 +386,14 @@ export class ToolTileComponent extends BaseComponent<IProps, IState> {
     const dragSrcContentId = getContentIdFromNode(model);
     if (!dragSrcContentId) return;
 
+    // dragging a tile selects it first
+    ui.setSelectedTile(model, { append: hasSelectionModifier(e) });
```

Comment:
🤦 Well, that's pretty obvious now that you point it out.

Reply:
Definitely wasn't very obvious to me initially.  It took a ton of console log statements and repeated drags to actually see what was going on.   Then the fix was incredibly clear.